front/planning.php

```diff
@@ -53,6 +53,10 @@
    $_GET["limititemtype"] = "";
 }
 
+if (isset($CFG_GLPI['planning_days'])) {
+   $CFG_GLPI['planning_days'] =  json_encode($CFG_GLPI['planning_days']);
```

Comment:
Values iun `$CFGGLPI` should not be altered. his is impossible to predict what kind of value we will get.